### PR TITLE
PROBLEM-3017 add skip fields

### PR DESCRIPTION
**Pull request by MickStanciu**
## Summary
Adds a new option to ignore certain fields. Right now, it works only for inspection_items / media_hypertext_reference

```
export:
  action:
    limit: 100
  asset:
    limit: 100
  incremental: true
  inspection:
    archived: "false"
    completed: "true"
    included_inactive_items: false
    limit: 100
    skip_ids: []
    web_report_link: private
  inspection_items:
    skip_fields: ["media_hypertext_reference"]
  issue:
    limit: 100
```

## Problem description

Certain customers have a huge number of files attached, causing trouble when exporting

## Checklist

- [x] Is this PR a reasonable size?

- [ ] List deployment sequence with all relevant PRs.
  - ...

---

**Code Review Guidelines** for Reviewers

- Try to review in a timely manner. Opinions/nitpicks should not be blockers. Pair on a call for non-trivial feedback.
- Overall design and approach should follow established patterns. Don't try to make the PR perfect.
- Try to identify edge cases, race conditions, over-engineering, lack of test coverage and complexity.
- If you don't feel qualified to review the code, pass it on to someone who is.

### CODE REVIEW

**Review by HarryTranAU — DISMISSED** [review]

File: pkg/internal/feed/feed_inspection_item.go [hunk 1]
```diff
@@ -237,6 +238,22 @@ func (f *InspectionItemFeed) writeRows(ctx context.Context, exporter Exporter, r
...
+// processSkipFields - will remove the content if the field is in the list.
+func processSkipFields(fields []string, row *InspectionItem) *InspectionItem {
+	if len(fields) == 0 {
+		return row
+	}
+
+	var rowClone = *row
+	for _, field := range fields {
+		switch field {
+		case "media_hypertext_reference":
```
Comment (sc-gv) [inline pkg/internal/feed/feed_inspection_item.go:250]: how about creating constants for this?

**Review by sc-gv — DISMISSED** [review]

File: pkg/internal/feed/feed_inspection_item.go (hunk 1 above)
Comment (MickStanciu) [reply pkg/internal/feed/feed_inspection_item.go:250]: makes sense: pushed

**Review by HarryTranAU — APPROVED** [review]
